src/components/TopBar.vue

```diff
@@ -61,10 +60,42 @@ const formatWalletBalance = (): String => {
           <span class="default-text"> Ethereum </span>
           <img alt="Chevron Down" src="@/assets/chevronDown.svg" />
         </div>
-        <div class="top-bar-info">
-          <img alt="Account image" src="@/assets/account.svg" />
-          <span class="default-text text-sm">{{ formatWalletAddress() }}</span>
-          <img alt="Chevron Down" src="@/assets/chevronDown.svg" />
+        <div class="flex flex-col">
+          <div class="top-bar-info" @mouseover="menuToggle = true" @mouseout="menuToggle = false" :style="{
```

Comment:
Add a cursor-pointer to this style class attr to show to the user that have a click action there